dockerfiles/Dockerfile.android-ci-base

```diff
@@ -37,5 +24,7 @@ COPY bugsnag-android-core/ bugsnag-android-core/
 COPY bugsnag-plugin-android-ndk/ bugsnag-plugin-android-ndk/
 COPY bugsnag-plugin-react-native/ bugsnag-plugin-react-native/
 COPY scripts/ scripts/
+COPY config/ config/
 
-RUN ./gradlew
+# Build and upload to the local maven as version 9.9.9
+RUN ./gradlew assembleRelease publishToMavenLocal -PVERSION_NAME=9.9.9 -PABI_FILTERS=arm64-v8a,armeabi,armeabi-v7a,x86,x86_64
```

Comment:
Assembling the bugsnag-android AAR here avoids the previous situation where it needed to be assembled twice in different containers